Mathlib/NumberTheory/Cyclotomic/Rat.lean

```diff
@@ -483,7 +483,7 @@ theorem not_exists_int_prime_dvd_sub_of_prime_pow_ne_two
         Nat.reduceAdd] at htwo ⊢
       exact htwo.symm.lt_of_le hp.1.two_le
     · exact one_lt_mul_of_lt_of_le (one_lt_pow₀ hp.1.one_lt hk)
-        (have := Nat.Prime.two_le hp.out; by cutsat)
+        (have := Nat.Prime.two_le hp.out; by omega)
```

Comment:
Does `cutsat` not work? I'd like Kim's feedback if so, though they may need some time to respond